pkg/components/external-dns/component.go

```diff
@@ -88,7 +88,10 @@ type component struct {
 	OwnerID        string    `hcl:"owner_id"`
 }
 
-func newComponent() *component {
+// NewConfig returns new ExternalDNS component configuration with default values set.
```

Comment:
```suggestion
// NewConfig returns new external-dns component configuration with default values set.
```

Reply:
`ExternalDNS` is the official form of the project name: https://github.com/kubernetes-sigs/external-dns#externaldns.

Reply:
Cool, then it needs to be fixed for the previous commit then

Reply:
Right, fixing.